itemManagement/views.py

```diff
@@ -47,6 +47,20 @@ def get(self, request, itemId, *args, **kwargs):
             return render(request, 'itemManagement/item_details_assistant.html', context=context)
 
 
+class ItemCreationView(TemplateView):
```

Comment:
admin only

Reply:
Remove ItemCreationView as this functionality will be added in a separate branch